components/infobox/wikis/starcraft/infobox_series_custom.lua

```diff
@@ -0,0 +1,205 @@
+	table.insert(widgets, Cell{name = 'Server', content = {_args.server}})
+	table.insert(widgets, Cell{name = 'Type', content = {_args.type}})
+	table.insert(widgets, Cell{name = 'Format', content = {_args.format}})
+	table.insert(widgets, Builder({
+		builder = function()
+			if _args.prizepooltot ~= 'false' then
+				return {
+					Cell{
+						name = 'Total prize money',
+						content = {CustomSeries._getSeriesPrizepools()}
+					}
+				}
+			end
+		end
+	}))
+
+	CustomSeries._addCustomVariables()
+
+	return widgets
+end
+
+function CustomInjector:parse(id, widgets)
+	if id == 'liquipediatier' then
+		return {
+			Cell{
+				name = 'Liquipedia tier',
+				content = {
+					CustomSeries._createLiquipediaTierDisplay(
+						_args.liquipediatier,
+						_args.liquipediatiertype or _args.tiertype
+					)
+				}
+			}
+		}
+	end
+
+	return widgets
+end
+
+function CustomSeries._getSeriesPrizepools()
+	local seriesTotalPrizeInput = Json.parseIfString(_args.prizepooltot or '{}')
+	local series = seriesTotalPrizeInput.series or _args.series or mw.title.getCurrentTitle().text
+
+	return SeriesTotalPrize._get{
+		series = series,
+		limit = seriesTotalPrizeInput.limit or _args.limit,
+		offset = seriesTotalPrizeInput.offset or _args.offset,
+		external = seriesTotalPrizeInput.external or _args.external,
+		onlytotal = seriesTotalPrizeInput.onlytotal or _args.onlytotal,
+	}
+end
+
+function CustomSeries._getPatch()
+	local patch = _args.patch
+	local endPatch = _args.epatch
+	if String.isEmpty(patch) then
+		return nil
+	end
+
+	Variables.varDefine('patch', patch)
+	Variables.varDefine('epatch', String.isNotEmpty(endPatch) and endPatch or patch)
+
+	if String.isEmpty(endPatch) then
+		return '[[' .. patch .. ']]'
+	else
+		return '[[' .. patch .. ']] &ndash; [[' .. endPatch .. ']]'
+	end
+end
+
+function CustomSeries._addCustomVariables()
+	if
+		(not Namespace.isMain()) or
+		_args.disable_smw == 'true' or
+		_args.disable_lpdb == 'true' or
+		_args.disable_storage == 'true'
+	then
+		Variables.varDefine('disable_SMW_storage', 'true')
+	else
+		--needed for e.g. External Cups Lists
+		local name = _args.name or mw.title.getCurrentTitle().text
+		Variables.varDefine('featured', _args.featured or '')
+		Variables.varDefine('headtohead', _args.headtohead or '')
+		Variables.varDefine('tournament_liquipediatier', _args.liquipediatier or '')
+		Variables.varDefine('tournament_liquipediatiertype', _args.liquipediatiertype or _args.tiertype or '')
+		Variables.varDefine('tournament_mode', _args.mode or '1v1')
+		Variables.varDefine('tournament_ticker_name', _args.tickername or name)
+		Variables.varDefine('tournament_shortname', _args.shortname or '')
+		Variables.varDefine('tournament_name', name)
+		Variables.varDefine('tournament_abbreviation', _args.abbreviation or _args.shortname or '')
+		Variables.varDefine('tournament_game', _args.game or '')
+		Variables.varDefine('tournament_type', _args.type or '')
```

Comment:
Should like they should use a Tournament Infobox or HDB and not a Series infobox?

Reply:
it is for series pages that use external cups lists
those store prize pool objects directly on the page of the series
the events on their own are not notable enough to warant their own pages but the series as a whole is notable, hence the results/prize pools are shown in a table form and stored on the page of the series

for those external cups lists the vars are needed (sc2 does the same btw)

example (from sc2) of such an external cups list:
![Screenshot 2022-05-25 12 45 40](https://user-images.githubusercontent.com/75081997/170245058-bd859530-c448-479f-8bbe-77ec4884c6b2.png)

Reply:
HDB seems more fitting then?

Reply:
they would need infobox series + hdb on those pages then
we could butcher the template to invoke both though

Reply:
Why do they need Infobox series?

Reply:
because it is a series page

or do you mean just not having the template at all and using infobox league everywhere?

Reply:
Trying to understand the needs are here exactly, why it's done as it is, and if there's a smarter way to do it

Reply:
We'll get it in for now and revisit later.